zng/literal.go

```diff
@@ -26,6 +26,9 @@ func ParseLiteral(literal ast.Literal) (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
+	if v.Type == nil {
+		return nil, nil
+	}
```

Comment:
Sorry I didn't catch it the first time around, but this can also move into the subsequent switch, as
```go
case nil:
	return nil, nil
```